giles-eco/src/main/java/edu/asu/diging/gilesecosystem/web/core/service/apps/impl/RegisteredAppsManager.java

```diff
@@ -53,10 +53,13 @@ public IRegisteredApp storeApp(IRegisteredApp app) {
         }
 
         try {
-            databaseClient.store(app);
+            databaseClient.storeRegisteredApp(app);
```

Comment:
if the dependency to databaseclient is removed, this probably does not have to be renamed